episodes/operations.md

```diff
@@ -27,28 +27,116 @@ By the end of this session, participants should be able to...
+](fig/cldt-structure.svg){
+alt="A graphical representation of the schedule and structure of collaborative lesson development training."
+width="67%"}
+
+To make scheduling easier,
+the parts of the training will be "decoupled" i.e.
+all groups of trainees attending the first part of the training 
+will not be required to also join the second part together.
+Groups of trainees collaborating on a lesson will still be required
+to attend each part together.
+
+![Groups of trainees collaborating on the same lesson will be able to
+choose which second part to attend,
+without needing to join the same session as the other groups they attended the first part with.
+](fig/decoupled-training-parts.svg){
+alt="diagram showing five groups of collaborators distributed across two different events teaching part 1 of Collaborative Lesson Development Training, with the same groups distributed differently across two part 2 training events."
+width="67%"
+}
+
+
+## Scheduling Trainings
+
+At time of writing, details of the system and processes we will use 
+to schedule CLDT events have not been finalised.
+Whatever the system looks like in the end,
+The Carpentries will try to schedule pairs of Trainers to teach parts 1 and 2 together,
+e.g. Trainer A teaches parts 1 & 2 alongside Trainer B,
+but this will depend on availability and sometimes we may need to mix up
+Trainers into different combinations
+e.g. Trainer A teaches part 1 with Trainer B, and part 2 with Trainer C.
+
+We will also aim to match the number of part 1 and part 2 events for any given timezone,
+aligning the gaps between parts to give the cohorts sufficient time to schedule and complete their Trial Runs.
+Trainees will only be able to sign up to join a part 2 event 
+after registering their Trial Run. 
+(Although they can register the Trial Run in advance i.e. it will not need to have happened yet).
+
+Like the system for scheduling trainings, 
+the system/process for Trial Run registration is still under development.
+In the training pilots, we used a relatively simple web form 
+but we may want to incorporate this data collection into AMY,
+to integrate it into the workflows for tracking trainee progress centrally.
+
+
+## What do Trainees require for certification?
+
+For a trainee to receive their Lesson Developer certificate,
+they will be required to:
+
+1. participate in both parts of Collaborative Lesson Development Training.
+2. teach a trial run of part of their lesson, and register that trial run with The Carpentries.
+
 :::::::::::::::::::::::::::::::::::::::::::::::::: discussion
 
 ## Discussion Questions
 
-- FIXME
+- This training is about lesson design and development skills, not teaching skills.
+  How important do you think it is that 
+  every member of a collaborative group should _have to_ teach during their trial run - 
+  as opposed to, say, one member teaching all of the groups' prepared episodes?
```

Comment:
Should this question mention that the other developers would still be required to attend/support the trial run?

Reply:
I thought about this but decided to leave the ambiguity, hoping to prompt discussion around whether it would be acceptable for one person to teach the Trial Run on their own and report back to their collaborators, or would everyone need to be there e.g. as helpers? What if one person was missing but the other three could attend? etc.

Reply:
That sounds like a great idea Toby!